components/prism-bqn.js

```diff
@@ -0,0 +1,65 @@
+Prism.languages.bqn = {
+	'shebang': {
+		pattern: /^#![ \t]*\/.*/,
+		alias: 'important',
+		greedy: true
+	},
+	'comment': {
+		pattern: /#.*/,
+		greedy: true
+	},
+	'string-literal': {
+		pattern: /"(?:[^"]|"")*"/,
+		greedy: true,
+		alias: 'string'
+	},
+	'character-literal': {
+		pattern: /'.'/,
```

Comment:
This is slightly wrong because of non-Unicode regexes. The docs say: "[Character literals](https://mlochbaum.github.io/BQN/doc/token.html#characters-and-strings) start and end with `'`, and have exactly one character in between." 

So one [Unicode character](https://mlochbaum.github.io/BQN/doc/types.html#characters). To correctly match this, we have to handle surrogate pairs.

```suggestion
		pattern: /'(?:[\s\S]|[\uD800-\uDBFF][\uDC00-\uDFFF])'/,
```

Reply:
Applied in recent patch.